frontend/package.json

```diff
@@ -12,7 +12,9 @@
     "test:update-snapshots": "jest --updateSnapshot",
     "watch": "jest --watch",
     "fix": "next lint --fix; prettier --write .",
-    "check-types": "tsc --noEmit"
+    "check-types": "tsc --noEmit",
```

Comment:
How did you set this up initially? Storybook init script? I'm surprised some of the transforms (sass) don't work

Reply:
yeah, me too - there's even a preset for Sass. but it kept blowing up - not just failing to display that one story (I think it was PrimaryButton) but breaking the Storybook build. apparently there were incompatibilities between versions of node, sass, and... something else, i can't remember. I bet we can get it working eventually, but for the short term, it seemed easier to just rewrite that one file as css and jettison the sass addon

Reply:
also, yes, I used `npx sb init` to start it up. but it was a little confusing because it was my third attempt, and I was pulling files in from two earlier branches, so it's possible it got a little jumbled